app.rb

```diff
@@ -0,0 +1,165 @@
+require_relative './book'
+require_relative './teacher'
+require_relative './student'
+require_relative './person'
+require_relative './rental'
+
+class App
+  def initialize()
+    @book_list = []
+    @rentals = []
+    @person = []
+  end
+
+  def menu
+    puts "Welcome to school library ! \n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
+    gets.chomp
+  end
+
+  def menu_run
+    choice = menu.to_s
+    case choice
+    when '1'
+      list_all_books
+    when '2'
+      list_all_person
```

Comment:
- [ ] Hi @ekenecf, great job for implementing the changes suggested by the previous reviewers :+1: . I think your app breaks after listing a person or a book. You will once more make sure that your app behaves like the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided to you.